### DISCUSSION TITLE
Components with semantic HTML5 tags?

### DISCUSSION THREAD

**Discussion by albertbronsky** [mantinedev/mantine#681] [Q&A] [answered]
Is using `<Box component="...">` the only way to use semantic HTML5 tags with Mantine components?

For example, if I would want to use `<Group>` component but have it as `<aside>` semantic tag in my HTML (instead of the default `<div>`) - what could be done to achieve this?

Because, writing everything in `<Box component="aside">` and manually re-writing the built-in behavior logic of `<Group>` component with `sx={{ display: 'flex', ... }}` seems like an overlap and anti-pattern?

**Comment by rtivital** [comment, marked as answer]
Wrap Group in aside:
```tsx
<aside>
  <Group>{/* ... */}</Group>
</aside>
```
